### PR TITLE
Poor man's profiling - custom solution

### PR DESCRIPTION
**Pull request by paterczm**
A very simple and unintrusive logging of crud operations, in lieu of proper profiling instrumentation. Main purpose is to be able to see inside bulk operations, but can be easily added to other lightblue modules and methods.

Warning: StopWatches on mediator operations will log entire requests, including secrets. For that reason this feature is disabled by default. It can be controlled using system properties (-Dstopwatch.enabled=true to start logging, -Dstopwatch.globalWarnThresholdMS=10000 to change global threshold).

### CODE REVIEW

**Comment by paterczm** [comment]
This is similar to https://github.com/lightblue-platform/lightblue-core/pull/792, except it offers some key improvements:
* can be disabled
* logging threshold can be changed without recompiling code
* will log time always in milliseconds, will not change units

**Comment by paterczm** [comment]
Added logic to check result size and log it if it exceeds threshold.

File: util/src/main/java/com/redhat/lightblue/util/JsonUtils.java [hunk 1]
```diff
@@ -443,6 +443,33 @@ private static void indent(StringBuilder bld, int depth) {
...
+        } else if (node instanceof ObjectNode) {
+            for (Iterator<Map.Entry<String, JsonNode>> fields = ((ObjectNode) node).fields(); fields.hasNext();) {
+                Map.Entry<String, JsonNode> field = fields.next();
+                size += field.getKey().length();
+                size += size(field.getValue());
+            }
+        } else if (node instanceof NumericNode) {
+            size += 4;
+        } else {
+            size += node.asText().length();
```
Comment (bserdar) [inline util/src/main/java/com/redhat/lightblue/util/JsonUtils.java]: Need nullcheck here

Comment (paterczm) [inline util/src/main/java/com/redhat/lightblue/util/JsonUtils.java]: Can a node ever be null (except when null is passed to JsonUtils.size)?

Comment (bserdar) [inline util/src/main/java/com/redhat/lightblue/util/JsonUtils.java]: No, but null can be passed to JsonUtils.size() in this context, and this is the only place it'll throw a NPE. Null was not a concern where this was copied from.

**Comment by bserdar** [comment]
+ return JsonUtils.size(response.getEntityData());

response.getEntityData() may be null here.

**Pull request merged**